api/api.proto

```diff
@@ -419,6 +419,33 @@ message Alerts {
   repeated Alert alert = 1;
 }
 
+// ObjectstoreInfo is a structure that has current objectstore info
+// swagger:model
+message ObjectstoreInfo {
+    // UUID of objectstore                                                      
+    string UUID = 1;                                                               
```

Comment:
This looks great, but for consistency make all variables lower case. The output of the api.pb.go will make them upper case.